<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dod_zlecenia</class>
 <widget class="QDialog" name="dod_zlecenia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodaj/usuń czynności</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1001</width>
     <height>501</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Wykonania</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_szukaj">
      <property name="title">
       <string>Lista czynności</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit_szukaj"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_szukaj">
         <item>
          <property name="text">
           <string>opis</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>idc</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>punkty</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_szukaj">
         <property name="text">
          <string>szukaj</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTableView" name="tableView_czynnosci"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_szukaj_wykonane">
      <property name="title">
       <string>Lista wykonań</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_szukaj_wykonane">
         <property name="text">
          <string>szukaj</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit_szukaj_wykonane"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_szukaj_wykonane">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>c.opis</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>c.punkty</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTableView" name="tableView_czynnosci_wykonane"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
